### PR TITLE
Add PyCon India 2021 Schedule Announcement blog

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

* **Please check if the PR fulfills these requirements**

- [ ] Screenshots for changes (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: content/blog/2021/2021-announce-final-schedule.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Title: We have liftoff! The final schedule is out!
+Date: 2020-08-02 17:00
```
Comment (raukadah) [inline content/blog/2021/2021-announce-final-schedule.md:2]: 2021

File: content/blog/2021/2021-announce-final-schedule.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Category: 2021
+Slug: announce-schedule
+Summary: PyCon India 2021 releases the schedule with a schedule packed with 30+ talks, 10+ workshops, four keynotes, and many more events for its next online iteration this year!
+
```
Comment (raukadah) [inline content/blog/2021/2021-announce-final-schedule.md:6]: 28 talks and 6 workshop

File: content/blog/2021/2021-announce-final-schedule.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+We thank everyone who submitted their ideas for talks, workshops, and poster sessions being organized this year. The call for proposals is a critical aspect of the conference, and we got over 150 submissions! The review committee, along with all the volunteers, worked quite hard to make the deadlines work. All these efforts go on to accomplish one outcome. An incredible PyCon India keeping you at its epicenter!
+
+As we are releasing the schedule for talk and keynotes, be sure to look out for the agenda for many more events in the coming days on [Twitter](https://twitter.com/pyconindia) and on [our mailing list](https://mail.python.org/mailman/listinfo/inpycon). Our rehearsal team is working double-time to get all the speakers in top shape for making this year's PyCon India a fantastic experience for everyone.
+
+**Checkout the [PyCon India 2021 schedule](https://in.pycon.org/2021/schedule/)**
```
Comment (raukadah) [inline content/blog/2021/2021-announce-final-schedule.md:13]: https://in.pycon.org/2021/#schedule

**Review by raukadah — CHANGES_REQUESTED** [review]
Looks good few wdits

File: content/blog/2021/2021-announce-final-schedule.md (hunk 3 above)
Comment (vipulgupta2048) [reply content/blog/2021/2021-announce-final-schedule.md:13]: Right on.

**Comment by vipulgupta2048** [comment]
Closes https://github.com/pythonindia/inpycon2021-tasks/issues/83

**Review by raukadah — APPROVED** [review]

**Comment by raukadah** [comment]
@anistark @RiteshAgrawal

**Review by anistark — APPROVED** [review]

**Pull request merged**